lib/phoenix/live_dashboard/page_builder.ex

```diff
@@ -793,21 +793,23 @@ defmodule Phoenix.LiveDashboard.PageBuilder do
 
   @doc false
   def live_dashboard_path(socket, route, node, old_params, new_params) when is_atom(node) do
-    apply(
-      socket.router.__helpers__(),
-      :live_dashboard_path,
-      if node == node() and is_nil(old_params["node"]) do
-        [socket, :page, route, new_params]
-      else
-        [socket, :page, node, route, new_params]
-      end
-    )
```

Comment:
I think we need to keep the previous code if we want v1.6 compatibility (which we need right now).